packages/react-devtools-core/webpack.backend.js

```diff
@@ -81,6 +81,22 @@ module.exports = {
       'process.env.LIGHT_MODE_DIMMED_ERROR_COLOR': `"${LIGHT_MODE_DIMMED_ERROR_COLOR}"`,
       'process.env.LIGHT_MODE_DIMMED_LOG_COLOR': `"${LIGHT_MODE_DIMMED_LOG_COLOR}"`,
     }),
+    {
+      apply: compiler => {
+        compiler.hooks.afterEmit.tap('AfterEmitPlugin', compilation => {
+          require('child_process').exec(
+            `
+            cp ~/react/packages/react-devtools-core/dist/backend.js ~/fbsource/xplat/js/public/node_modules/react-devtools-core/dist/
```

Comment:
Should this be in here?

Reply:
Yes, this + the `typeof Proxy === undefined` changes are there only for convenience of local testing. I'll remove both before committing.